javascript.js

```diff
@@ -23,3 +25,270 @@ document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click'
   navbarLinks.classList.remove('active');
   title.classList.remove('active');
 }));
+
+// Dynamic projects section
+
+// define array for storing  object for card content
+const arrayElements = [];
+
+// define objects  with content for each project card
+const main_project = {
+  title: 'Multi-Post Stories', description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industries standard dummy textever since the 1500s, when an unknown printer took a standard dummy text.', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder.png',
+};
+const project1 = {
+  title: 'Website Portfolio', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-2.png',
+};
+const project2 = {
+  title: 'Data Dashboard', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-3.png',
+};
+const project3 = {
+  title: 'Brand website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-4.png',
+};
+const project4 = {
+  title: 'Shopping Website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-2.png',
+};
+const project5 = {
+  title: 'School Website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-3.png',
+};
+const project6 = {
+  title: 'eLearning Website', description: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima voluptatibus saepe quae dolore', features: ['HTML', 'CSS', 'Javascript'], imageURL: 'images/Img_Placeholder-4.png',
+};
+
+// add objects to array
+arrayElements.push(project1);
+arrayElements.push(project2);
+arrayElements.push(project3);
+arrayElements.push(project4);
+arrayElements.push(project5);
+arrayElements.push(project6);
```

Comment:
- [ ] instead of doing it like you can loop through them and add all of them in one line so that will avoid DRY don't repeat yourself